doc/lsst.ap.verify/running.rst

```diff
@@ -67,7 +69,7 @@ The command for running the pipeline on Gen 3 data is almost identical to Gen 2:
 
 .. prompt:: bash
 
-   ap_verify.py --dataset HiTS2015 --gen3 --id "visit in (412518, 412568) and band='g'" --output workspaces/hits/
+   ap_verify.py --dataset CI-CosmosPDR2 --gen3 --id "visit in (59150, 59160) and band='g'" --output workspaces/cosmos/
```

Comment:
To keep Gen3 example parallel with Gen2 example, consider adding a second example with no `--id` argument.